docs/Contributing/API-for-contributors.md

```diff
@@ -3349,6 +3351,27 @@ Install self-service software on macOS, Windows, or Linux (Ubuntu) host. The sof
 
 `Status: 202`
 
+### Uninstall software via self-service
+
+Uninstalls software from a host via the My device page.
+
+`POST /api/v1/fleet/device/{token}/software/uninstall/:software_title_id`
```

Comment:
```suggestion
`POST /api/v1/fleet/device/:token/software/uninstall/:software_title_id`
```

Reply:
@rachaelshaw We use `{}` in contributor docs. Should I create a PR to change this across the board?